README.md

```diff
@@ -0,0 +1,195 @@
+
+Learn more about Rudder features and use cases [on the website](https://www.rudder.io/).
+
+### Online demo
+
+Browse the [demo Web interface](https://demo.rudder.io) (with fake data) to get a glimpse of the available features and interfaces.
+
+<a href="https://demo.rudder.io"><img height="300" alt="Rudder dashboard" src="readme-resources/dashboard.png" /></a>
+
+### Components
+
+A Rudder installation is made of:
+
+* A central server, providing the Web interface, the [HTTP API](https://docs.rudder.io/api) and the automation engine. It can be extended with plugins.
+* (optional) Relays acting as smart proxies between the server and the agents.
+* A light agent installed on every managed system. It runs autonomously and check and/or modifies the state of the system continuously (in practice, every 5 minutes by default).
+
+## Install Rudder
+
+> [!TIP]
+> You need an active subscription to get a complete setup, including Rudder's security features. Without a subscription
+> you will get access to [Rudder Core](#rudder-core) only.
+> It you want to evaluate how Rudder can help you, you can [ask for a free trial](https://www.rudder.io/free-trial/) on the website.
+>
+> If you have installed Rudder Core, you can upgrade it at any time to a subscription
+> or a free trial license.
+
+### Standard installation
+
+Follow the step-by-step instructions of the documentation to setup Rudder:
+
+* 📥 [**Install Rudder**](https://docs.rudder.io/reference/current/installation/index.html), with or without a subscription.
+* You can then follow the [get started documentation](https://docs.rudder.io/get-started/current/configuration-policies/index.html) to discover the features with a hands-on tutorial.
+
+### Quick installation
+
+Alternatively, for a quick setup on Linux systems, you can use the installer script.
+For a Rudder server:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://repository.rudder.io/tools/rudder-setup | sh -s -- setup-server latest
+```
+
+For a Rudder agent, replace `SERVER` by you Rudder server's IP or hostname:
+
+```bash
+curl --proto '=https' --tlsv1.2 -sSf https://repository.rudder.io/tools/rudder-setup | sh -s -- setup-agent latest SERVER
+```
+
+For more options and to review the script before running it:
+
+```bash
+wget https://repository.rudder.io/tools/rudder-setup
+chmod +x rudder-setup
+./rudder-setup -h
+```
+
+## FAQ
+
+
+### Is Rudder open-source? Is it free? What is Rudder Core?
+
+Rudder has adopted an _open-core_ model:
+
+* **Rudder Core** is available for free and totally open-source (under GPL 3.0 an Apache 2.0). It includes the main Rudder components, Linux agents, plus several plugins.
+* The complete **Rudder** solution is paid and partially open-source. A part of the paid plugins and agents are distributed under proprietary license. It also includes other benefits:
+
+- Additional plugins that add major features
```

Comment:
having a shift for the second list would make it clearer (with 4 spaces before -)